TestGetContainerStateAfterUpdate is failing on CI

```
=== RUN   TestGetContainerStateAfterUpdate
--- FAIL: TestGetContainerStateAfterUpdate (0.00s)
    container_linux_test.go:363: unimplemented
```

https://travis-ci.org/github/opencontainers/runc/jobs/669942726?utm_medium=notification&utm_source=github_status

--- a/libcontainer/container_linux_test.go
+++ b/libcontainer/container_linux_test.go
@@ -15,10 +15,11 @@ import (
 )
 
 type mockCgroupManager struct {
-	pids    []int
-	allPids []int
-	stats   *cgroups.Stats
-	paths   map[string]string
+	pids        []int
+	allPids     []int
+	stats       *cgroups.Stats
+	paths       map[string]string
+	unifiedPath string
 }
 
 type mockIntelRdtManager struct {
@@ -55,7 +56,7 @@ func (m *mockCgroupManager) GetPaths() map[string]string {
 }
 
 func (m *mockCgroupManager) GetUnifiedPath() (string, error) {
-	return "", fmt.Errorf("unimplemented")
+	return m.unifiedPath, nil
 }
 
 func (m *mockCgroupManager) Freeze(state configs.FreezerState) error {

